src/components/ShowMenus/ShowMenus.js

```diff
@@ -7,13 +8,15 @@ export default function ShowMenus({menus, curClickedCategory}) {
         .filter(menu => curClickedCategory === 'All' || menu.category === curClickedCategory)
```

Comment:
Just a personal preference: I like to do the filter before the `return` statement and not right before the `map`.

Reply:
Do you have examples? Here the logic is, filter menus that I don't want to be rendered, then render. First version is map then filter,  I had missing key error, because every element in map must have a key, even if I render <></>, I must give this thing a key, to give this a key, I need a div, div effect the layout.

Reply:
What I mean is the following:
```js
export default function ShowMenus({menus, curClickedCategory}) {
  const filteredMenus = menus.filter(menu => curClickedCategory === 'All' || menu.category === curClickedCategory)

  return (
    <MenusContainer>
      {filteredMenus
        .map(menu => {
```
To have the `filter` just before rendering :)